Test/UnitTesting.cpp

```diff
@@ -47,7 +47,7 @@ UnitTesting& UnitTesting::instance()
 bool UnitTesting::runTest(vstring const& unitId, vstring const& testCase) 
 {
   auto unit = findUnit(unitId);
-  if (unit == nullptr) return false;
+  if (unit == nullptr) return true;
```

Comment:
I'm not sure whether that's the right thing to do. How can i reproduce the issue that u wanna solve with that?
Only adding the `#if VZ3` does the job for compiling and testing everything without `z3` on my machine.

I think the `return false;` should stay so we get a test failure if we try to run a non-existing test case.

Reply:
ah sorry just found what the issue you meant was. I'll have a quick look and give a new comment then.

Reply:
I'd propose to let the `return false;`s stay `return false;`s, and skipping the z3 tests in the makefile instead. I made these changes and pushed them.

Reply:
Thanks for the change, I agree that it's a better solution. There is probably no way anyway to make sure no one adds new tests that would fail without Z3.